app/Http/Controllers/CityController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\City;
+use Illuminate\Http\Request;
+
+class CityController extends Controller
+{
+    public function index(){
+
+        return view('cities.show',[
+            'links' => City::orderby('id','asc')->select('*')->paginate()->links()
+        ]);
+    }
+
+    public function store(Request $request){
+
+        $attributes = $request->validate([
+            'name' => 'required|unique:cities'
+        ]);
+
+        $city = City::create($attributes);
+        return Response()->json($city);
+    }
+
+    public function getCities(){
+        $cities = City::orderby('id','asc')->select('*')->paginate();
```

Comment:
```suggestion
        $cities = City::paginate();
```
The default order is ASC by ID, so you can remove the `orderby('id','asc')` (which has a typo, btw, it's `orderBy`). `SELECT *` is the default, so you can remove `select('*')`